README.md

```diff
@@ -1 +1,184 @@
-# Tesla-car-booking-front-end
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h2><b>Tesla Car Booking Backend</b></h2>
+  <h4>Ivan Gonzalez & Bruno Kambere & Ali Baba & Hafiz Rasa & Steven Wafeek</h4>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🔗 Link to Frontend React app](#link-frontend)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [📑 Database ERD](#erd)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Tesla Car Booking Frontend <a name="about-project"></a>
+
+**Tesla Car Booking Frontend** is the back-end app that provides a user api for booking Tesla cars test drives. This app is integrated with the front end React app linked below.
+
+## 💻 Link to front end React app <a name="link-frontend"></a>
+
+The link to the React app:
+
+- [React app link](https://github.com/ivangonzalez224/Tesla-car-booking-front-end)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Stack</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Api endpoints**
+- **Api endpoints documentation**
+- **User Authorization && Authentication**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📑 Database ERD <a name="erd"></a>
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍